hack/quickstart/kubelet.master

```diff
@@ -1,6 +1,6 @@
 [Service]
 Environment=KUBELET_IMAGE_URL=quay.io/coreos/hyperkube
-Environment=KUBELET_IMAGE_TAG=v1.5.3_coreos.0
+Environment=KUBELET_IMAGE_TAG=v1.5.6_coreos.0
```

Comment:
We shouldn't bump the quickstart version until we have an official release (because it uses the bootkube container image). So this would affectively just update the quickstart kubelet, but not the rest of the components. This is pretty confusing and I'd like to just change quickstart to use locally compiled binaries: https://github.com/kubernetes-incubator/bootkube/issues/399#issuecomment-291346075

That being said - after we cut a release, we should bump the quickstart guide too (it has fallen behind).